civihr_default_theme/scss/base/_variables.scss

```diff
@@ -22,6 +22,8 @@ $light:               #ffffff !default; // well bg, text on dark bg hover
 $light-pale:          #c3cbd6 !default; // text on dark bg
 $light-gray:          #eeeded !default;
 
+$text-muted:          rgba(118, 123, 143, 0.6); // #757b8e with 60% opacity
```

Comment:
can you try using `rgba($gray, 0.6)` instead?

Reply:
Yes, that worked, thank you.